data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
I'm still struggling to follow this a little. ISTM that the field that Pombola internally calls the slug is really the outward facing identifier, as it's what's used in the URLs, and if you're planning to also keep the URLs the same on the new site, then I think that this is definitely OK as a plain `identifier__shineyoureye`.

I _think_ that the internal Pombola IDs are a bit of a red herring here, though I'm open to persuasion that there's some value in EP having them. If there is, then I think we should come up with a different name for them to convey how someone would actually consume/use them.